architecture/crypto.rst

```diff
@@ -191,9 +191,51 @@ GP requirements tested and covered by the OP-TEE sanity test suite
 supported - i.e: the SE05x does not implement all RSA key sizes - we opted for
 disabling those particular tests at build time rather than letting them fail.
 
+NXP SE05X Family of Secure Elements
+***********************************
+
+This family of I2C bus devices are supported through the se050 cryptographic driver
+located at `core/drivers/crypto/se050`_. Before the REE boots, the session with
+the device is established using one of the OP-TEE supported I2C platform device
+drivers. Once the REE is up, the cryptographic driver can be configured to use
+the I2C driver in the REE (via RPC service) or continue using the one in OP-TEE.
+
+Unless the Secure Element owns the I2C bus (no other elements on the bus, no
+runtime-PM and so forth), it is recommended to route all traffic via the Normal
+World. Initial communication with the device is not data intensive and therefore
+slow I2C drivers - perhaps those not using DMA channels - do not represent much
+of a performance drag; the situation changes once clients start hammering the
+device.
+
+If using the REE for I2C transfers, it is also recommended to configure the
```

Comment:
s/also recommended/imperative/ ? I mean without protection in place it's worse than useless, right?

Reply:
yes, absolutely; as important as this one: 

https://optee.readthedocs.io/en/latest/architecture/porting_guidelines.html#trusted-application-private-public-keypair 

will use the same tone as in the link above.

Reply:
@jenswi-linaro updated

Reply:
Looks good.